server/views/applications/edit.njk

```diff
@@ -0,0 +1,50 @@
+{% extends "_layouts/default.njk" %}
+
+{% block main_content %}
+
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+  <a href="/manage-applications" class="govuk-back-link">Back to all applications</a>
+  <h1 class="govuk-heading-xl">
+    My test application
+  </h1>
+  <h2 class="govuk-heading-l">
+    Update application
+  </h2>
+  <div class="govuk-form-group">
+    <label class="govuk-label" for="event-name">
```

Comment:
I understand these are just placeholders but I think using "event-name" for all label attachments is confusing.

Reply:
I assume when we come to do the "real data" tickets the names will change